data/korath/korath missions.txt

```diff
@@ -577,3 +577,19 @@ mission "Efreti: Distress Call 8"
 			`	You return to your ship and try plugging the chip into your systems again, which now displays two files: one in the Korath language, and one in your own. You open the file in your language, and it reads: "The <EDC-ship> is a ship of grief, bringing great pain in memories that sing for generations. Love is a void in the souls of many. Know all that <first> <last> is a true Efret by heart, if not by skin. The <EDC-ship> is theirs, to save lives and keep memories."`
 			`	You carefully store the chip in a safe place.`
 
+
+mission "Exile: Boarding Worldship Korath"
+	boarding
+	minor
+	destination "Earth"
+	source
+		government "Korath" "Korath (Civilian)"
+		category "Superheavy"
+		outfits "Afterburner (Stellar Class)"
+	on offer
+		conversation
+			`Docking with the gargantuan vessel allows you a front-row seat to witness the sheer scale of this ship. Your sensors detect tens of thousands of life-signs aboard, along with numerous other minor lifesigns, perhaps the trees that you can see in the massive dome atop the craft?`
+			`	As you scan the ship, your computer advises you that a ship of this scale would require an actual army and likely weeks of constant fighting to even have a chance of fully capturing it. A thorough sweep would be essential to preventing the Korath from fighting guerrilla campaigns against you, even after you claim it.`
```

Comment:
I feel like the addition of "even if you hired enough crew to man the many hundreds of essential stations in the ship," is just words for words sake. It might just be me but other than that I like all the other additions.

Reply:
If the emphasis is on actually manning the ship, something like "even if you could afford to hire enough crew to man the many hundreds of essential stations in the ship," since it's currently not obvious that's the implication.
Though I also think referring to how many people the player can get doesn't make sense, since human space has tens of billions of people to go grab crew for and the player often is rich when they have alien tech anyway.

Reply:
Well, that's why I had it as "even if" not "even if you could." Many players indeed could, but it's still a massive logistical jump from the totals of any other individual ship they have be paying to man so far.

It totally works as current without that bit though, so probably best to leave it out unless you or someone else comes up with a better way to slide it back in :)